composer.json

```diff
@@ -9,6 +9,11 @@
         "php": ">=5.3.3"
     },
 
+    "require-dev": {
+        "php-yaoi/php-yaoi": "1.0.18",
```

Comment:
Btw, as it seems you are the maintainer of this package, it would deserve getting a README update to better document it, as it seems there is no doc at all